contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -74,6 +75,8 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   error InvalidRecipient();
   error InvalidDataLength();
   error TargetCheckReverted(bytes reason);
+  error OnlyActiveUpkeep();
```

Comment:
There's also an error `UpkeepNotActive`. Can we merge those / clear the semantics ?

Reply:
Done